### DISCUSSION TITLE
File .svx format

### DISCUSSION THREAD

**Discussion by githubfan1** [Keriew/augustus#704] [General]
About saved files .svx, are you going to disclose the file format? I am thinking about tools to edit the saved files as was done with the original .sav files. ty

**Comment by crudelios** [comment]
Hello!

Well, the file format is more or less disclosed in the code. Other that that, it's basically adapted from the original c3.

There's an issue for the editor, though: since we often add features, we need to constantly change the file format, so I don't think it's feasible to have an editor right now.
